cmd/storage-rest-server.go

```diff
@@ -486,6 +486,65 @@ func (s *storageRESTServer) RenameFileHandler(w http.ResponseWriter, r *http.Req
 	}
 }
 
+// Send whitespace to the client to avoid timeouts as bitrot verification can take time on spinning/slow disks.
+func sendWhiteSpaceVerifyFile(w http.ResponseWriter) <-chan struct{} {
```

Comment:
Can we use sendWhiteSpace and avoid making multiple functions for the same reason?

Reply:
sendWhiteSpace has:
```
                                // Write header if not written yet.
                                if !headerWritten {
                                        w.Write([]byte(xml.Header))
                                        headerWritten = true
                                }
```

and comments there are heavily tailored for XML/CompleteMultipart related behavior.
Better to keep it as two separate functions.

Reply:
Okay